addons/op-badge/userscript.js

```diff
@@ -0,0 +1,17 @@
+export default async function({ addon, console }) {
+    const posts = document.querySelectorAll(".blockpost");
+
+    const op = await fetch(`https://scratchdb.lefty.one/v3/forum/topic/posts/${window.location.href.split("/")[5]}/0?o=oldest`).then(res => res.json());
+
+    for (const i of posts) {
+        if (i.querySelector(".username").innerText == op[0].username) {
+            let br = document.createElement("br");
+            let op_badge = document.createElement("span");
+
+            op_badge.innerText = "OP";
```

Comment:
I think "Original poster" should be used instead of "OP". Also, all addon string should be [localized](https://scratchaddons.com/docs/localization/localizing-addons/).

Reply:
We could also use the word "Author", like GitHub does.

Reply:
Also see https://scratchaddons.com/docs/develop/userscripts/#localizing-userscripts